Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM python:3.8.5-slim-buster
```

Comment:
So we are going to be using 3.8? Asking so I can use the right version in my local virtual env

Reply:
Good point. I've updated the image. It's 3.9 actually. That is the latest stable version.